### PR TITLE
Frontend config

### PR DESCRIPTION
**Pull request by mehdi-zibout**
* Add Typography config
* Add React Router
* Add React Query

### CODE REVIEW

**Review by theborgh — APPROVED** [review]
Looks good!

File: frontend/index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
       rel="stylesheet"
     />
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="icon" type="image/svg+xml" href="/logo.svg" />
```
Comment (poly4concept) [inline frontend/index.html:11]: @mehdi-zibout I have noticed on some projects I worked on recently that this favicon might not show in some browsers(safari to be specific). 
The solution I found is to use this [website](https://realfavicongenerator.net/) to generate the specific supported images for each browser.

We can always work on this later

**Review by poly4concept — APPROVED** [review]
Amazing work so far.

File: frontend/index.html (hunk 1 above)
Comment (mehdi-zibout) [reply frontend/index.html]: This is a good remark, Thank you.
I did the necessary changes.

**Pull request merged**